internal/imports/check-imports.go

```diff
@@ -0,0 +1,37 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build ignore
+
+package main
+
+import (
+	"log"
+	"os"
+
+	"gonum.org/v1/gonum/internal/imports"
+)
+
+var (
```

Comment:
This is a single var, so it doesn't need a paren block (same in the test file).

Reply:
done